lively.morphic/text/morph.js

```diff
@@ -13,7 +13,6 @@ import { Range } from './range.js';
 import { eqPosition, lessPosition } from './position.js';
 import KeyHandler from '../events/KeyHandler.js';
 import { UndoManager } from '../undo.js';
-import { TextSearcher } from './search.js';
```

Comment:
🔖 Bookmark for myself, can be safely ignored :slightly_smiling_face: 
Please do not resolve this!